types/budget.d.ts

```diff
@@ -40,6 +42,15 @@ declare global {
         budget: number;
       }
 
+      export interface Options {
+        /**
+         * Origins that should be considered first-party.
+         * If not supplied, only the origin of the main resource will be considered first-party.
```

Comment:
Won't we consider all of the root domain to be first-party right now? 

If we're planning on changing that behavior when this launches we should probably flag it as a separate breaking change.